pyiron/base/generic/hdfio.py

```diff
@@ -1285,7 +1285,13 @@ def to_object(self, object_type=None, **qwargs):
         new_obj = self.create_object(obj_type, **qwargs)
         if obj_type != str(type(new_obj)):  # Backwards compatibility
             self["TYPE"] = str(type(new_obj))
-        new_obj.from_hdf()
+        if obj_type != "<class 'pyiron.atomistics.structure.atoms.Atoms'>":
```

Comment:
If I understand correctly, for all objects except `Atoms`, we are loading it from the directory where `TYPE` is written right? And for `Atoms` we go one step back. I really like the way we do it for the `Atoms` class because it seems more intuitive. Should we make this consistent for all classes or are there some restrictions?

Reply:
Yes, in the mid-term we should make it more consistent. For now it is more like a quick fix to simplify loading atomistic structures.